packages/console/src/ds-components/Button/index.tsx

```diff
@@ -122,25 +125,42 @@ export function LinkButton({
   type = 'default',
   size = 'medium',
   className,
+  href,
   targetBlank,
   ...rest
 }: LinkProps) {
-  return (
-    <a
-      className={classNames(styles.button, styles[type], styles[size], className)}
-      {...rest}
-      {...(Boolean(targetBlank) && {
+  const { getTo } = useTenantPathname();
+  const props = useMemo(
+    () => ({
+      ...rest,
+      className: classNames(styles.button, styles[type], styles[size], className),
+      ...(Boolean(targetBlank) && {
         rel: typeof targetBlank === 'string' ? targetBlank : 'noopener noreferrer',
         target: '_blank',
-      })}
-    >
-      {typeof title === 'string' ? (
+      }),
+    }),
+    [className, rest, size, targetBlank, type]
+  );
+
+  const innerElement = useMemo(
+    () =>
+      typeof title === 'string' ? (
         <span>
           <DynamicT forKey={title} />
         </span>
       ) : (
         title
-      )}
+      ),
+    [title]
+  );
+
+  return !href || isAbsoluteUrl(href) ? (
+    <a {...props} href={href}>
+      {innerElement}
     </a>
+  ) : (
+    <Link {...props} to={getTo(href)}>
```

Comment:
Looks like not all the LinkProps are supported by the LinkButton wrapper. e.g. `replace`, and `state`.

Reply:
The Button component will be refactored in another PR. Linear task created
https://linear.app/silverhand/issue/LOG-7011/refactor-button-component-in-admin-console